### ISSUE TITLE
New Analytics API in Jira

### ISSUE THREAD

**Issue by AGhost-7** [EFForg/privacybadger#2358]
Noticed that some new endpoints appear to be sending data over to `segment.io` when using Jira, which isn't being blocked by privacy badger:

`https://*.atlassian.net/gateway/api/gasv3/api/v1/p`
![Screen Shot 2019-05-07 at 12 01 35 PM](https://user-images.githubusercontent.com/6957411/57314569-e97f2880-70bf-11e9-8974-d24e2f24a091.png)

`https://*.atlassian.net/gateway/api/gasv3/api/v1/t`
![Screen Shot 2019-05-07 at 12 03 05 PM](https://user-images.githubusercontent.com/6957411/57314658-16cbd680-70c0-11e9-9da6-7607197df214.png)

**Comment by bcyphers** [comment]
Is the issue here that Privacy Badger should be blocking the requests? If so, can you check whether "segment.io" is present in your snitch map at all? (just step 3 [here](https://github.com/EFForg/privacybadger/wiki/Find-out-why-Privacy-Badger-is-blocking-a-domain#3-copy-the-following-javascript-and-paste-it-into-the-background-page-console))

**Comment by AGhost-7** [comment]
Yes, the issue is that I think Privacy Badger should be blocking those requests. It doesn't look like `segment.io` is in the map.

**Comment by bcyphers** [comment]
Possibly related: #367

**Comment by ghostwords** [comment]
Looks like these are POST requests. Related to #794, which is probably about looking into GET request query strings.

**Comment by ghostwords** [comment]
If we were to implement this, we'd probably want to ignore user-initiated form submissions (https://github.com/EFForg/privacybadger/issues/2221#issuecomment-442134054).

**Comment by ghostwords** [comment]
I think this is part of Atlassian's Global Analytics Service (GASv3).

https://www.atlassian.com/engineering/cloud-overview

>GASv3 exposes an API that is used by all of our product analytic client libraries to transmit structured data into our pipeline.

So then this is first party tracking but maybe integrated into third parties on the backend?

Does `segment.io` show up anywhere in your browser's network tools when these "gasv3" requests are sent?

How would Privacy Badger know that these are tracking requests?
